xcodeproj/internal/top_level_targets.bzl

```
@@ -314,6 +315,11 @@ The xcodeproj rule requires {} rules to have a single library dep. {} has {}.\
 
     cc_info = target[CcInfo] if CcInfo in target else None
     objc = target[apple_common.Objc] if apple_common.Objc in target else None
+    codesignopts = getattr(ctx.rule.attr, "codesignopts", None)
```

Can we add `codesignopts` to `XcodeProjAutomaticTargetProcessingInfo`, and then use this instead?
```suggestion

    codesignopts = getattr(
        ctx.rule.attr,
        automatic_target_info.codesignopts,
        None,
    )
```